Engine/main/graphics_mode.cpp

```diff
@@ -91,9 +91,7 @@ DisplayModeSetup::DisplayModeSetup()
 
 Size get_desktop_size()
```

Comment:
It's worth trying if you can remove this function and use platform->GetDesktopSize everywhere directly.

Reply:
Ah, good point, should be cleaner this way. I will do this in a separate commit later.